sqlglot/tokens.py

```diff
@@ -383,6 +384,7 @@ class Tokenizer(metaclass=_Tokenizer):
         "FUNCTION": TokenType.FUNCTION,
         "FOLLOWING": TokenType.FOLLOWING,
         "FOREIGN KEY": TokenType.FOREIGN_KEY,
+        "FORMAT": TokenType.FORMAT,
```

Comment:
whenever you add a token you need to add it to the ID_VAR_CONSTANTS or else you won't be able to use it as an alias 

select x format

Reply:
Thanks!